### DISCUSSION TITLE
blog/refine-dev-backend

### DISCUSSION THREAD

**Discussion by giscus** [zenstackhq/zenstack-docs#320] [Comments]
# blog/refine-dev-backend

This post introduces how to build a fully-secured backend API for refine.dev projects with minimum code using Prisma and ZenStack.

https://zenstack.dev/blog/refine-dev-backend

**Comment by AlexanderRyzhov** [comment]
Do I feel right that there is a typo in **Automatic CRUD API** section?
API route handler shoud be created by path `src/app/api/model/[...path]/route.ts`, not `src/app/model/[...path]/route.ts` as shown in code-block section header?

**Reply by jiashengguo** [reply]
Thanks, fixed it.  That's really a hard one to notice.
